frontend/packages/kubevirt-plugin/src/components/vmtemplates/vmtemplates.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
```

Comment:
Nit: please use `src/components/vm-templates` directory and `vm-template.tsx` module to stick with existing name conventions.

Reply:
Cool :+1: